### PR TITLE
Win build

### PR DESCRIPTION
**Pull request by ocefpaf**
Continuation of #5 

@msarahan and @mingwandroid I appreciate if you can review this.

This also updates the version, but most of our software is pinned which means we should be OK. (Brave last words :wink:)

PS: I am not sure if `msys2` needs some sort of run-time dependency.

### CODE REVIEW

**Comment by mingwandroid** [comment]
`m2-*` is never a runtime dependency. `m2w64-*` can be, but only if MSVC can't be used to compile the thing in question.

Personally, I would squash nearly all of these commits, leaving the following:
```
"Windows build with MSYS2" (contains everything up to:)
"Fix race condition making out/tmp/icudat.res"
"rerender"
```

**Comment by ocefpaf** [comment]
> `m2w64-*` can sometimes be, but only if MSVC can't be used to compile the thing in question (e.g. the `R` ecosystem).

If I use `MSYS2` to compiled Fortran extension would that be true as well?

> Personally, I would squash nearly all of these commits, leaving the following:

Sure. Makes sense.

**Comment by mingwandroid** [comment]
If `m2w64-toolchain` is used to make a fortran DLL then yes, you need to add `m2w64-gcc-libs` as a runtime dependency.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -2,71 +2,67 @@
...
 extra:
-  recipe-maintainers:
-    - ccordoba12
-    - jakirkham
-    - msarahan
+    recipe-maintainers:
+        - ccordoba12
+        - jakirkham
+        - msarahan
+        - ocefpaf
```
Comment (mingwandroid) [inline recipe/meta.yaml]: Can you add me to this please? I'm not sure what the deal is here, does it happen automatically, or is it up to the person themselves or to be done by someone already trusted with maintainence of the package?

**Comment by ocefpaf** [comment]
> If `m2w64-toolchain` is used to make a fortran DLL then yes, you need to add `m2w64-gcc-libs` as a runtime dependency.

Thanks. I will experiment with that soon.

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipe/meta.yaml]: > Can you add me to this please?

Of course!

> I'm not sure what the deal is here, does it happen automatically, or is it up to the person themselves or to be done by someone already trusted with maintainence of the package?

All that is needed is a PR adding your GitHub handle (either me now or you later). Once merged you get access to the feedstock.

**Comment by jakirkham** [comment]
Sorry this is going a bit off topic, but is there a more granular way to do libraries, @mingwandroid? For instance, is there a `m2w64-gfortran-libs`?

**Comment by mingwandroid** [comment]
@jakirkham : Yes, there is, but I don't bother since we don't bother on other OSes. There we have `libgcc` and I'm fine with that. The libraries aren't that big.

**Comment by ocefpaf** [comment]
@msarahan and @mingwandroid should we merge this?

**Comment by msarahan** [comment]
Sure, fine by me.  I can't say I've looked deeply at it, though.  If it works (and the CIs seem to say that it does) then go for it.

File: appveyor.yml [hunk 2]
```diff
@@ -20,22 +20,8 @@ environment:
...
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 34
-
-    - TARGET_ARCH: x86
-      CONDA_PY: 35
-
-    - TARGET_ARCH: x64
-      CONDA_PY: 35
```
Comment (ocefpaf) [inline appveyor.yml]: This does not look OK :confused:

File: recipe/bld.bat [hunk 3]
```diff
@@ -1,15 +1,39 @@
...
+set MSYSTEM=MINGW%ARCH%
+set MSYS2_PATH_TYPE=inherit
+set CHERE_INVOKING=1
+:: Set flag saying we have stdint.h manually.  We take care of providing it where
+::    it is needed, but missing, by using msinttypes
+set "CXXFLAGS=%CXXFLAGS% -DU_HAVE_STDINT_H=1"
+set "CFLAGS=%CFLAGS% -DU_HAVE_STDINT_H=1"
+
+:: Set PATH to include msys2's binaries
+set "PATH=%PATH%;%LIBRARY_PREFIX%\usr\bin;%LIBRARY_PREFIX%\mingw-w64\bin"
```
Comment (mingwandroid) [inline recipe/bld.bat]: This is no longer necessary, but shouldn't be harmful really.

**Comment by ocefpaf** [comment]
Closing in favor of #15

**Pull request closed without merging**